app/presenters/search_presenter.rb

```diff
@@ -11,7 +11,11 @@ def to_json
       description: document.summary,
       link: document.base_path,
       indexable_content: indexable_content,
-      organisations: organisation_slugs,
+      # FIXME: This line was dependent on a method called #organisation_slugs,
+      # which in turn used publishing_api.get_content to map org content_ids to
+      # organisation slugs. It was stubbed in many places in spec/, but that method
+      # call always returned a blank list in the real world.
+      organisations: [],
```

Comment:
Rummager uses the organisations tag to 1) allow filtering by organisation in search and 2) display a metadata thingy in the search result:

<img width="1176" alt="screen shot 2016-05-12 at 12 45 48" src="https://cloud.githubusercontent.com/assets/233676/15213599/2b9ad196-1840-11e6-95f0-18de136896aa.png">